libraries/wutdevoptab/devoptab_fsa_utils.cpp

```diff
@@ -0,0 +1,267 @@
+#include <cstdio>
+#include "devoptab_fsa.h"
+#include "../wutnewlib/wut_clock.h"
+
+
+#define COMP_MAX      50
+
+#define ispathsep(ch) ((ch) == '/' || (ch) == '\\')
```

Comment:
Are backslash paths actually supported on Wii U?

Reply:
Backslashs are not supported, but `__wut_fsa_normpath` converts them to `/`